PROJECT-CLEARWATER-MIB

```diff
@@ -5050,6 +5050,80 @@ chronosInvalidTimersProcessedCount OBJECT-TYPE
     DESCRIPTION "The total number of invalid timers processed over the period."
     ::= { chronosInvalidTimersProcessedEntry 2 }
 
+chronosAllTimersTable OBJECT-TYPE

```

Comment:
@rkd-msw changed the MIB file to fix up various warnings in https://github.com/Metaswitch/clearwater-snmp-handlers/pull/93. Can you make the equivalent changes to this, and run smilint over the result.